modules/ximgproc/include/opencv2/ximgproc/fast_line_detector.hpp

```diff
@@ -0,0 +1,109 @@
+/*********************************************************************
+ * Software License Agreement (BSD License)
+ *
+ * Copyright (c) 2014, 2015
+ * Zhengqin Li <li-zq12 at mails dot tsinghua dot edu dot cn>
+ * Jiansheng Chen <jschenthu at mail dot tsinghua dot edu dot cn>
+ * Tsinghua University
```

Comment:
Oops... Because I wrote the code based on LSD,, I made such a mistake. I will change it right away.